agent/engine/docker_container_engine.go

```diff
@@ -571,10 +571,12 @@ func (dg *dockerGoClient) ContainerEvents(ctx context.Context) (<-chan DockerCon
 
 	go func() {
 		for event := range events {
-			containerId := event.ID
-			if containerId == "" {
+			// currently only container events type needs to be handled
+			if event.Type != "container" || event.ID == "" {
```

Comment:
@hasimo The minimum required version of Docker to use with the ECS agent is 1.5.0.